test/unit/uk/gov/hmrc/bindingtariffclassification/controllers/CaseControllerSpec.scala

```diff
@@ -92,6 +93,7 @@ class CaseControllerSpec extends UnitSpec with WithFakeApplication with MockitoS
     "return 201 when the case has been created successfully" in {
       when(caseService.nextCaseReference).thenReturn(successful("1"))
       when(caseService.insert(any[Case])).thenReturn(successful(c1))
+      when(caseService.addInitialSampleStatusIfExists(any[Case])).thenReturn(Future.successful(():Unit))
 
```

Comment:
If the controller/service implementation stays the same then maybe some tests that the addInitialSampleStatusIfExists` service method is/isn't called when required.